main.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func setupFlags() {
 	flag.StringVar(&config.Host, "host", "0.0.0.0", "host for the dice server")
-	flag.IntVar(&config.Port, "port", 7379, "port for the dice server")
+	flag.IntVar(&config.Port, "port", 6379, "port for the dice server")
```

Comment:
Do not change the port. The default port of DiceDB is 7379.

Reply:
the file was added mistakely, sorry for that